src/ngraph/op/util/unary_elementwise_arithmetic.hpp

```diff
@@ -44,6 +44,13 @@ namespace ngraph
             class UnaryElementwiseArithmetic : public Op
             {
             protected:
+                UnaryElementwiseArithmetic();
```

Comment:
I think this is another one that snuck in- sorry, should have flagged it on last review.

Reply:
Evicted. I don't see any more.